app/src/main/java/com/celzero/bravedns/ui/HomeScreenActivity.kt

```diff
@@ -235,7 +228,6 @@ class HomeScreenActivity : AppCompatActivity(R.layout.activity_home_screen) {
     private fun showNewFeaturesDialog() {
         if (checkToShowNewFeatures()) {
             updateInstallSource()
-            updateForDownload()
```

Comment:
what is this change diff for?

Reply:
This is specific for the previous version. The updateFoeDownload() is not required for upcoming versions so removed.